developer/contributing.md

```diff
@@ -32,3 +30,45 @@ Let us know how we can make the contribution process easier. We want to find the
 -   Should passing a linter code review and follow Reaction style guidelines.
 -   Contributors should review the CLA.
 -   Code reviewed before merge acceptance.
+
+### Here's what to expect when you make a pull request to Reaction
+
+As soon as pull requests are pushed, [BitHound](https://www.bithound.io/github/reactioncommerce/reaction), [CircleCI](https://circleci.com/gh/reactioncommerce/reaction) and [Synk](https://snyk.io/) run automated tests. All checks should pass successfully to ensure:
+
+-   All linting rules pass without errors or ignores.
+-   Dependencies are not failing.
+-   All tests pass.
+-   No new security vulnerabilities are introduced.
+
+You can run these checks yourself by running these commands:
+
+-   `eslint .` for linting
+-   `reaction test` for all tests
+
+### Core team pull request review
```

Comment:
The whole "core team review" is no longer true. Probably best to just say that the Community team will review all community PRs as soon as possible. All of these notes apply equally to both Core and Community PR's.